completions/zsh/_swaylock

```diff
@@ -4,56 +4,59 @@
+  '(--layout-border-color)'--layout-border-color'[Sets the color of the border of the box containing the layout text]:color:' \
+  '(--layout-text-color)'--layout-text-color'[Sets the color of the layout text]:color:' \
+  '(--line-caps-lock-color)'--line-caps-lock-color'[Sets the color of the line between the inside and ring when Caps Lock is active]:color:' \
+  '(--line-clear-color)'--line-clear-color'[Sets the color of the line between the inside and ring when cleared]:color:' \
+  '(--line-color)'--line-color'[Sets the color of the line between the inside and ring]:color:' \
+  '(--line-uses-inside -n)'{--line-uses-inside,-n}'[Use the inside color for the line between the inside and ring]' \
+  '(--line-uses-ring -r)'{--line-uses-ring,-r}'[Use the ring color for the line between the inside and ring]' \
+  '(--line-ver-color)'--line-ver-color'[Sets the color of the line between the inside and ring when verifying]:color:' \
+  '(--line-wrong-color)'--line-wrong-color'[Sets the color of the line between the inside and ring when invalid]:color:' \
+  '(--no-unlock-indicator -u)'{--no-unlock-indicator,-u}'[Disable the unlock indicator]' \
+  '(--ring-caps-lock-color)'--ring-caps-lock-color'[Sets the color of the ring of the indicator when Caps Lock is active]:color:' \
+  '(--ring-clear-color)'--ring-clear-color'[Sets the color of the ring of the indicator when cleared]:color:' \
+  '(--ring-color)'--ring-color'[Sets the color of the ring of the indicator]:color:' \
+  '(--ring-ver-color)'--ring-ver-color'[Sets the color of the ring of the indicator when verifying]:color:' \
+  '(--ring-wrong-color)'--ring-wrong-color'[Sets the color of the ring of the indicator when invalid]:color:' \
+  '(--scaling -s)'{--scaling,-s}'[Image scaling mode: stretch, fill, fit, center, tile, solid_color]:mode:(stretch fill fit center tile solid_color)' \
+  '(--separator-color)'--separator-color'[Sets the color of the lines that separate highlight segments]:color:' \
+  '(--show-failed-attempts -F)'{--show-failed-attempts,-F}'[Show current count of failed authentication attempts]' \
+  '(--show-keyboard-layout -k)'{--show-keyboard-layout,-k}'[Display the current xkb layout while typing]' \
+  '(--text-caps-lock-color)'--text-caps-lock-color'[Sets the color of the text when Caps Lock is active]:color:' \
+  '(--text-clear-color)'--text-clear-color'[Sets the color of the text when cleared]:color:' \
+  '(--text-color)'--text-color'[Sets the color of the text]:color:' \
+  '(--text-ver-color)'--text-ver-color'[Sets the color of the text when verifying]:color:' \
+  '(--text-wrong-color)'--text-wrong-color'[Sets the color of the text when invalid]:color:' \
+  '(--tiling -t)'{--tiling,-t}'[Same as --scaling=tile]' \
+  '(--version -v)'{--version,-v}'[Show the version number and quit]' \
+  '(--verstr)'--verstr'[The string displayed when the password is being verified.]:string:' \
+  '(--wrongstr)'--wrongstr'[The string displayed when the password is wrong.]:string:'
```

Comment:
Please never mix formatting and functional changes in a single commit. And don’t introduce any formatting changes here.